datamodel/low/base/schema_test.go

```diff
@@ -415,23 +415,23 @@ examples:
 	mbErr := low.BuildModel(rootNode.Content[0], &sch)
 	assert.NoError(t, mbErr)
 
-    schErr := sch.Build(rootNode.Content[0], nil)
-    assert.NoError(t, schErr)
-    assert.Equal(t, "something object", sch.Description.Value)
-    assert.Len(t, sch.Type.Value.B, 2)
-    assert.True(t, sch.Type.Value.IsB())
-    assert.Equal(t, "object", sch.Type.Value.B[0].Value)
-    assert.True(t, sch.ExclusiveMinimum.Value.IsB())
-    assert.False(t, sch.ExclusiveMinimum.Value.IsA())
-    assert.True(t, sch.ExclusiveMaximum.Value.IsB())
-    assert.Equal(t, float64(12), sch.ExclusiveMinimum.Value.B)
-    assert.Equal(t, float64(13), sch.ExclusiveMaximum.Value.B)
-    assert.Len(t, sch.Examples.Value, 1)
-    assert.Equal(t, "testing", sch.Examples.Value[0].Value)
-    assert.Equal(t, "fish64", sch.ContentEncoding.Value)
-    assert.Equal(t, "fish/paste", sch.ContentMediaType.Value)
-    assert.True(t, sch.Items.Value.IsB())
-    assert.True(t, sch.Items.Value.B)
+	schErr := sch.Build(rootNode.Content[0], nil)
+	assert.NoError(t, schErr)
+	assert.Equal(t, "something object", sch.Description.Value)
+	assert.Len(t, sch.Type.Value.B, 2)
+	assert.True(t, sch.Type.Value.IsB())
+	assert.Equal(t, "object", sch.Type.Value.B[0].Value)
+	assert.True(t, sch.ExclusiveMinimum.Value.IsB())
+	assert.False(t, sch.ExclusiveMinimum.Value.IsA())
+	assert.True(t, sch.ExclusiveMaximum.Value.IsB())
+	assert.Equal(t, float64(12), sch.ExclusiveMinimum.Value.B)
+	assert.Equal(t, float64(13), sch.ExclusiveMaximum.Value.B)
+	assert.Len(t, sch.Examples.Value, 1)
+	assert.Equal(t, "testing", sch.Examples.Value[0].Value)
+	assert.Equal(t, "fish64", sch.ContentEncoding.Value)
+	assert.Equal(t, "fish/paste", sch.ContentMediaType.Value)
+	assert.True(t, sch.Items.Value.IsB())
+	assert.True(t, sch.Items.Value.B)
```

Comment:
Would you be able to add an assertion in here to cover the const?

Reply:
done

Reply:
LGTM!